Makefile

```diff
@@ -122,8 +122,8 @@ deploy: manifests kustomize
 	$(KUSTOMIZE) edit set image ghcr.io/kedacore/keda=${IMAGE_CONTROLLER}
 	cd config/metrics-server && \
     $(KUSTOMIZE) edit set image ghcr.io/kedacore/keda-metrics-apiserver=${IMAGE_ADAPTER}
-	cd config/default && \
-    $(KUSTOMIZE) edit add label -f app.kubernetes.io/version:${VERSION}
+	@sed -i".out" -e 's@version:[ ].*@version: $(VERSION)@g' config/default/kustomize-config/metadataLabelTransformer.yaml
```

Comment:
Could you please add comment on this Makefile target to explain why do we need to do this workaround, ie. link this issue in it: https://github.com/kubernetes-sigs/kustomize/issues/1009

Reply:
sure comments added.